W/WIGXJPF/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WIGXJPF"
+version = v"1.13.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://fy.chalmers.se/subatom/wigxjpf/wigxjpf-latest.tar.gz", "90ab9bfd495978ad1fdcbb436e274d6f4586184ae290b99920e5c978d64b3e6a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/wigxjpf-1.13
+
+mkdir -p $WORKSPACE/destdir/lib
+mkdir -p $WORKSPACE/destdir/bin
+mkdir -p $WORKSPACE/destdir/shared/licenses
+
+if [ "$(uname)" == "Darwin" ]; then
+    make lib/libwigxjpf_shared.dylib
+    cp lib/libwigxjpf_shared.dylib $WORKSPACE/destdir/lib/  
+else
+    make lib/libwigxjpf_shared.so
+    cp lib/libwigxjpf_shared.so $WORKSPACE/destdir/lib/
+    cp lib/libwigxjpf_shared.so $WORKSPACE/destdir/bin/libwigxjpf_shared.dll                    
+fi
+
+cp README $WORKSPACE/destdir/shared/licenses/LICENSE
```

Comment:
```suggestion
install_license COPYING COPYING.LESSER
```

Please install the proper license file.